client/src/main/main.go

```diff
@@ -4,42 +4,73 @@ import (
 	"log"
 	"os"
 	"github.com/pborman/getopt/v2"
+	"bytes"
+	//"fmt"
+	//"strings"
 
-	client "../client"
+	client "project_aurum/client/src/client"
```

Comment:
This import path is causing my file to fail to compile. Is the old version doing the same for you?

Reply:
It doesn't do it for me, but that's likely because of where my project is installed. That's just an artifact from the branch talking about using absolute import paths rather than relative. But I'll change it back.

Reply:
The branch is now updated